docs/Guide/preconditions/reporting-precondition-failure.mdx

```diff
@@ -0,0 +1,241 @@
+about the context in which the error was thrown, and the value can be absolutely anything.
+
+We can take advantage of this by passing the value `{ silent: true }` to the [`this.error()`][preconditionError] call of
+a function. We'll use the `OwnerOnly` precondition we made in [Creating Preconditions][creating-preconditions] to
+demonstrate this.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {9}
+const { Precondition } = require('@sapphire/framework');
+
+module.exports = class OwnerOnlyPrecondition extends Precondition {
+  public run(message) {
+    return message.author.id === 'YOUR_ID'
+      ? this.ok()
+      : this.error({ 
+          message: 'Only the bot owner can use this command!' 
+          context: { silent: true }
+        })
+  }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {9}
+import { Precondition } from '@sapphire/framework';
+
+export class OwnerOnlyPrecondition extends Precondition {
+  public run(message) {
+    return message.author.id === 'YOUR_ID'
+      ? this.ok()
+      : this.error({ 
+          message: 'Only the bot owner can use this command!',
+          context: { silent: true }
+        })
+  }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {10}
+import type { Message } from '@sapphire/framework';
+import { Precondition } from '@sapphire/framework';
+
+export class OwnerOnlyPrecondition extends Precondition {
+  public run(message: Message) {
+    return message.author.id === 'YOUR_ID'
+      ? this.ok()
+      : this.error({ 
+          message: 'Only the bot owner can use this command!'
+          context: { silent: true }
+        })
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+We can then detect this property in our listener, and ignore the failure if we find it.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript {5}
+const { Listener } = require('@sapphire/framework');
+
+module.exports = class CommandDeniedListener extends Listener {
+    run(error, { message }) {
+        if (error.context.silent) return;
+        return message.channel.send(error.message);
+    }
+};
+```
+
+</TabItem>
+<TabItem value="ESM" label="ESM">
+
+```javascript {5}
+import { Listener } from '@sapphire/framework';
+
+export class CommandDeniedListener extends Listener {
+    run(error, { message }) {
+        if (error.context.silent) return;
+        return message.channel.send(error.message);
+    }
+}
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript {8}
+import type { UserError, CommandDeniedPayload } from '@sapphire/framework';
+import { Listener } from '@sapphire/framework';
+
+export class CommandDeniedListener extends Listener {
+    public run(error: UserError, { message }: CommandDeniedPayload) {
```

Comment:
context is undefined
```suggestion
    public run({ message , context}: CommandDeniedPayload) {
```

Reply:
Sorry, it should've been `error.context`. (`context` isn't actually in `CommandDeniedPayload`, nor would it be the first parameter)

Reply:
Then fix it however you want and resolve this conflix also error,  { dest} won't work because error is a single argument not two, am I right @favna

Reply:
https://github.com/Spectera-bot/Spectera/blob/main/src/listeners/commands/commandDenied.ts#L4